charm/kafka/reactive/kafka.py

```diff
@@ -139,7 +120,7 @@ def get_package_version(snap_name):
     return None
 
 
-@when('kafka.started', 'zookeeper.ready')
+@when('kafka.started', 'zookeeper.joined')
```

Comment:
These states are not the same thing, I'm now wondering if this breaks something.